backend/geonature/core/auth/routes.py

```diff
@@ -112,6 +112,15 @@ def loginCas():
                 "id_organisme": organism_id,
             }
             response.set_cookie("current_user", str(current_user), expires=cookie_exp)
+
+            # Log the user in
+            user = db.session.execute(
```

Comment:
Consider using a try-except block when querying the database to handle potential exceptions and provide a more informative error message to the user. [important]